Makefile

```diff
@@ -0,0 +1,15 @@
+.PHONY: build
+build:
```

Comment:
can you use the full command I pasted previously. I think the `verify=<code_hash>` is a neat little thing.